zos_concepts/rest_apis/README.md

```diff
@@ -0,0 +1,172 @@
+
+
+## Ansible Collection Requirement
+
+* IBM z/OS core collection 1.2.0 or later
+
+## Playbook Summary
+
+- [**uri-sample.yml**](uri-sample.yml) - Provides an example of how to invoke the z/OSMF REST API services with Ansible.
+
+## Getting Started
+
+If you are unfamiliar with playbooks, you can review our
+[detailed configuration guide](https://github.com/IBM/z_ansible_collections_samples/blob/master/docs/share/configuration_guide.md) or
+continue with getting started below.
+
+Optionally, you can use the sample
+[host_setup](https://github.com/IBM/z_ansible_collections_samples/blob/master/zos_administration/host_setup/README.md)
+to discover and create your **inventory** and **host_vars** artifacts. It should
+be noted that when you use the **host_setup** it will generate a configuration
+for the most common dependencies, some playbooks require more customized
+configurations, in this case, you can review the sample documentation and
+add the additional required variables.
+
+### Update [inventories/zos_host](inventories/zos_host) with the information about your system(s)
+
+```yaml
+# the system where the data should be copied to
+source_system:
+  hosts:
+    zos_host:
+      ansible_host: zos_target_address
+      ansible_user: zos_target_username
+      ansible_python_interpreter: path_to_python_interpreter_binary_on_zos_target
+```
+
+### Update the environment variables for each z/OS system in [host_vars/zos_host.yml](host_vars/zos_host.yml)
+
+```yaml
+# the path to the root of IBM python installation
+PYZ: "/usr/lpp/IBM/cyp/v3r8/pyz"
+
+# the path to root of ZOAU installation
+ZOAU: "/usr/lpp/IBM/zoautil"
+```
+
+### Update the playbook's [group_vars](group_vars/all.yml) with the corresponding values
+
+These variables are used within the `uri-sample` playbook and are required for using the z/OSMF REST API.
+
+```yaml
+# z/OSMF host and port information
+ZOSMF_HOST: zosmf_target_host
+ZOSMF_PORT: zosmf_target_port
+
+# Desired destination folder on target host for holding definition file
+ZOSMF_DEST_FOLDER: /path/to/folder
+
+# Credentials to access z/OSMF
+ZOSMF_USER: user
+ZOSMF_PASS: pass
+
+# Nickname of the system on which the workflow is to be created.
+# Use the nickname that is specified for the system definition
+# in the z/OSMF Systems task.
+ZOSMF_SYSTEM: zosmf_system
+```
+
+### Walkthrough of uri-sample.yml
+
+The purpose of the `uri-sample` playbook is to showcase the ability to invoke a REST API service as well as give an example of creating, starting, and deleting workflows using the z/OSMF REST API service.
+
+
+This playbook will use the following operations:
+* Querying existing workflows
+* Creation of a workflow
+* Starting a workflow
+* Cancellation of a workflow
+* Deletion of a workflow
+
+**Note:** to learn more about z/OSMF workflow services, visit the [knowledge center](https://www.ibm.com/docs/en/zos/2.4.0?topic=services-zosmf-workflow)
+
```

Comment:
Knowledge Center is now IBM Documentation or IBM Docs. Please fix the text

Reply:
Done :)